test/Microsoft.ML.Tests/ScenariosWithDirectInstantiation/TensorflowTests.cs

```diff
@@ -310,6 +316,7 @@ public void TensorFlowTransformInputShapeTest()
                 }
                 Assert.False(cursor.MoveNext());
             }
+            (tfModel as IDisposable).Dispose();
```

Comment:
❗️ This doesn't handle null correctly. One of the following should be used:

```suggestion
            (tfModel as IDisposable)?.Dispose();
```

```suggestion
            ((IDisposable)tfModel).Dispose();
```

Reply:
In this case, tfModel is TensorFlowModel which implements IDisposable. So it should always be non-null. But I have added the ? check in case the pipeline above changes in the future. 

---
In reply to: [396132941](https://github.com/dotnet/machinelearning/pull/4939#discussion_r396132941) [](ancestors = 396132941)